app/scripts/lib/constants.js

```diff
@@ -127,9 +127,9 @@ module.exports = {
 
   UTM_SOURCE_EMAIL: 'email',
 
-  // Recovery keys are base32 encoded, length 28 gives 135 bits of entropy
-  // Ex. (28 char - 1 version char) * 5 bits = 135 bits
-  RECOVERY_KEY_LENGTH: 28,
+  // Recovery keys are base32 encoded, length 32 gives 155 bits of entropy
+  // Ex. (32 char - 1 version char) * 5 bits = 155 bits
```

Comment:
Tiny nit, but I think this comment could read a bit clearer. 31 * 5 gives the size of the value but the entropy of that value is 31^32 or 2^155, i.e. there is a 1 in 2^155 probability of two random recovery keys clashing. Maybe that's supposed to be obvious from the "bits of entropy" part of the previous sentence, but if we're putting an equation in too I think it would be clearer if that much larger value was the result.